### ISSUE TITLE
Slack links are not descriptive enough

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-design-system#895]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**: [2.4.9 Link Purpose (Link Only) (Level AAA)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)
**Issue ID**: DAC_Issue2
**URL**: https://design-system.service.gov.uk/

## Summary
Links were encountered that were not descriptive to some users.

## Screen Shot
![](https://user-images.githubusercontent.com/2445413/58792743-b19ad080-85ec-11e9-9928-2c370cb858c3.png)

There was a non-descriptive link present that screen reader users did not understand when navigating out of context.

## Current Code Ref(s)
```html
<li> <a href="https://ukgovernmentdigital.slack.com/messages/govuk-design-system" class="govuk-link" data-hsupport="slack"> get in touch on Slack</a> (<a class="govuk-link" href="slack://channel?team=T04V6EBTR&amp;id=C6DMEH5R6" data-hsupport="slackapp">open in app</a>) </li>
```

## Screen reader comments
> “As I navigated out of context, I discovered a link which read to me as “Open in app.” It was not immediately clear to me while situated out of context which app the link related to as this was not obvious within the link text. Although I could establish the purpose of the link in context in a testing environment some users might not be aware of which app is intended to open. Including the app name within the link text will enable users to clearly understand what will occur if the link is selected. This was present using JAWS, NVDA, VoiceOver for iPhone and Mac”

## Solution
Include the ‘open in app’ text within the same link as the ‘get in touch on Slack’ text to ensure that screen reader users can understand the context of the link.

**Comment by nickcolley** [comment]
@36degrees thinks there might be away to consolidate the two links into one

**Comment by 36degrees** [comment]
So, we could update the links to use https://ukgovernmentdigital.slack.com/app_redirect?channel=govuk-design-system and remove the separate 'open in app' link.

If the user is logged into the ukgovernmentdigital workspace, then it'll _try_ and open in the Slack app, but also provides a link to download the app or to open in the browser:

<img width="1392" alt="Screen Shot 2019-06-05 at 15 58 16" src="https://user-images.githubusercontent.com/121939/58966766-ccb33f00-87aa-11e9-900c-4751ff811b79.png">

However, if the user's not logged in, they'll just get a login screen:

<img width="1392" alt="Screen Shot 2019-06-05 at 15 57 55" src="https://user-images.githubusercontent.com/121939/58966799-d8066a80-87aa-11e9-8273-3e3801cf744f.png">

This simplifies the links, but arguably provides a less optimal experience for users who are signed in in the app, but not in their browser.

**Comment by nickcolley** [comment]
Confirmed as fixed in a re-test by DAC on 7th October 2019
